logbus/formatter/formatter.go

```diff
@@ -261,7 +261,18 @@ func (f *Formatter) getCommand(commandID string) *command {
 }
 
 func (f *Formatter) handleDeltaLog(dl *logstream.DeltaLog) error {
-	c, verboseOnly := f.targetConsole(dl.GetTargetId(), dl.GetCommandId())
+	commandID := dl.GetCommandId()
+	targetID := dl.GetTargetId()
+
+	//lookup --raw-output from the command manifest
+	cm := f.manifest.GetCommands()[dl.GetCommandId()]
+	rawOutput := false
+	if cm != nil {
+		//commandStr building order in converter.internalRun means
+		// --raw-output is always first after run
+		rawOutput = strings.Contains(cm.Name, "RUN --raw-output")
```

Comment:
By adding the new manifest do you mean adding to logstream.DeltaLog? yeah, I can do it that way. Do I just raise a PR on cloud for the proto?

Reply:
I actually suggested using the `strings.Contains(cm.Name, "--raw-output")` approach as the initial v1 implementation instead of publishing a new proto, which seems overkill given there's no use-case for having this data on the sent up to our cloud logs.

An alternative approach was to add some sort of map on the Formatter struct, which we could check here.

Maybe yet a different approach would be to simply change the interface of the formattedSubs list, to allow passing along the vertexmeta struct.

Reply:
Sorry, I meant the "manifest field".

Reply:
For most cases, we've been extending the protobufs for manifest settings. Mind, you this is purely for display so it may be overkill.
 
> no use-case for having this data on the sent up to our cloud logs

I believe all formatted logs are sent up to the server. https://github.com/earthly/earthly/blob/main/logbus/setup/setup.go#L105

Using a new property would also allow us to identify (and perhaps hide) these lines in the UI.